client/stylesheets/controls.styl

```diff
@@ -7,32 +7,32 @@
 
 #ga-map-credit-box
   position absolute
-  left 0
-  bottom 0
+  left 10px
+  bottom 10px
   font-size 10px
-  margin-right 5px
-  margin-bottom 2px
+  padding 4px
   background rgba(255,255,255,0.5)
   z-index 1
 
+.ga-main-ui
+  width 335px
+
 .ga-main-ui,.ga-results-ui
   user-select none
   position absolute
   .panel
-    background none
+    background-color rgba(255,255,255,0.75)
   .panel-title
     font-size 14px
     color black
   .panel-heading
     padding 3px 5px
-    background-color rgba(255,128,0,0.75)
   .panel-default
     border 0
   .panel-body
     padding 5px
-    background-color rgba(255,160,160,0.75)
   .panel-group
-    margin-bottom 1px
+    margin 10px
```

Comment:
I liked the smaller gaps between panels and around the edges...  It left more room in the center.  I don't feel strongly about it, but I figured I'd comment.